core/src/main/scala/org/ensime/indexer/SearchService.scala

```diff
@@ -137,7 +137,7 @@ class SearchService(
     def findBases(): (Set[FileObject], Map[FileName, Set[FileObject]]) = {
       val (jarFiles, dirs) = config.modules.flatMap {
         case (name, m) =>
```

Comment:
The match is on a `id`, you can rename it for consistency, even though its not relevant here. You can also ignore it with `_`